pyanaconda/payload/dnf/payload.py

```diff
@@ -822,10 +823,23 @@ def base_repo(self):
         """Get the identifier of the current base repo or None."""
         # is any locking needed here?
         repo_names = [constants.BASE_REPO_NAME] + constants.DEFAULT_REPOS
+        subscription_proxy = SUBSCRIPTION.get_proxy()
```

Comment:
This will fail on Fedora. You should reorganize the code and make sure that we can access the Subscription module.

Reply:
Good, point! Changed! :)

Reply:
Please, fix also the RHEL 8 branch. It will fail on Cent OS as well.

Reply:
Good point, I'll do that based on the final form of this PR.